src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -707,50 +708,43 @@ public class Program
         Microsoft.CodeAnalysis.Runtime.Instrumentation.FlushPayload();
     }
 }
-";
-            string expectedOutput = @"Flushing
-Method 3
-File 1
-True
-True
-Method 4
-File 1
-True
-True
-Method 5
-File 1
-True
-True
-True
-True
-True
-Method 6
-File 1
-True
-True
-True
-Method 8
-File 1
-True
-True
-False
-True
-True
-True
-True
-True
-True
-True
-True
-True
-True
-True
-";
-
-            CompilationVerifier verifier = CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.ReleaseExe);
+" + InstrumentationHelperSource;
+
+            var checker = new CSharpInstrumentationChecker();
+            checker.Method(3, 1, "public int Prop3")
+                .T("get");
```

Comment:
Speaking of readability, it took some time to realise `T` and `F` stand for True and False. Could we use True and False instead.